java/feathub-udf/flink-udf/src/main/java/com/alibaba/feathub/flink/udf/aggregation/sum/SumAggFunc.java

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2022 The Feathub Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.feathub.flink.udf.aggregation.sum;
+
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.common.typeinfo.Types;
+import org.apache.flink.table.api.DataTypes;
+import org.apache.flink.table.types.DataType;
+
+import com.alibaba.feathub.flink.udf.aggregation.AggFunc;
+
+/** Aggregation functions that calculates the sum of values. */
+public abstract class SumAggFunc<IN_T>
```

Comment:
Since packages like `sum` only have 1-2 classes, would it be simpler to put all these classes under the package `com.alibaba.feathub.flink.udf.aggregation`?